File: sdks/ruby/lib/cyberarmor/providers/meta.rb
# frozen_string_literal: true

module CyberArmor
  module Providers
    class Meta < OpenAICompatible
      def initialize(client:, openai:, tenant_id:, model: 'llama-3.3-70b-instruct')
        super(client: client, openai: openai, tenant_id: tenant_id, provider: 'meta', model: model)
      end
    end
  end
end

File: sdks/php/src/Providers/Meta.php
<?php

declare(strict_types=1);

namespace CyberArmor\Providers;

use CyberArmor\Client;

final class Meta extends OpenAICompatible
{
    public function __construct(Client $client, object $openai, string $tenantId, string $model = 'llama-3.3-70b-instruct')
    {
        parent::__construct($client, $openai, $tenantId, 'meta', $model);
    }
}

